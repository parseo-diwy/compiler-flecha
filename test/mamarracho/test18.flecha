---- Primitiva: AND

def printBool x =
  case x
  | True      -> (unsafePrintChar 'T'; unsafePrintChar '\n')
  | False     -> (unsafePrintChar 'F'; unsafePrintChar '\n')

def colgarse x = colgarse x

def main =
  printBool (True  && True)  ;
  printBool (True  && False) ;
  printBool (False && True)  ;
  printBool (False && False) ;
  unsafePrintChar '-'; unsafePrintChar '\n';
  printBool (True && colgarse 0)


