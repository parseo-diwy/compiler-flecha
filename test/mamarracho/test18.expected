T
T
T
F
-
T
